kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -31,7 +34,14 @@ def __init__(self, model_part, params, file_header):
         self.__ValidateAndAssignOutputFolderPath()
 
         # size of the buffer in bytes. Set to "0" for flushing always
-        self.write_buffer_size = params["write_buffer_size"].GetInt()
+        is_mpi_execution = (model_part.GetCommunicator().TotalProcesses() > 1)
+        if not is_mpi_execution and not self.has_initial_write_buffer_size:
+            info_msg  = "File output buffer size set to 1 \n"
+            info_msg += "for ComputeDragProcess output file "+ self.file_name
+            KratosMultiphysics.Logger.PrintInfo("ComputeDragProcess", info_msg)
+            file_handler_params["write_buffer_size"].SetInteger(1)
```

Comment:
I think this should be `self.write_buffer_size = 1` (otherwise `self.write_buffer_size` is not set)

(BTW SetInt(1) would work here)